src/server_manager/web_app/app.ts

```diff
@@ -354,9 +354,12 @@ export class App {
         name: await this.digitalOceanAccount.getName(),
       };
       const status = await this.digitalOceanAccount.getStatus();
-      if (status !== digitalocean.Status.ACTIVE) {
+      if (status.needsBillingInfo || status.needsEmailVerification) {
         return [];
       }
+      if (status.warning) {
+        this.showDigitalOceanWarning(status.warning);
```

Comment:
I wonder if this should happen before you return aboove

Reply:
OK, I've achieved that effect by relaxing the checks here.  This function now returns servers if it is able to find any, and also surfaces a warning or error if necessary.